games-emulation/libretro-nestopia/libretro-nestopia-0.0.1_pre20180723.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LIBRETRO_REPO_NAME="libretro/nestopia"
+LIBRETRO_COMMIT_SHA="9c335527c5dfa11dfa29bc22ee4ba6efa23c8890"
+KEYWORDS="~amd64 ~x86"
+inherit libretro-core
+S="${S}/libretro"
```

Comment:
Don't use S om S, this will go crazy if ebuild is sourced multiple times.

Reply:
Are you saying that I shouldn't use S within S? If that's true, that `S="${WORKDIR}/${P}/libretro"` would be okay, so I've done that.